resource/reactivedrop_german.txt

```diff
@@ -386,31 +386,31 @@
 "[english]asw_name_flynn"		"Flynn"
 "asw_name_vegas"		"Vegas"
 "[english]asw_name_vegas"		"Vegas"
-"asw_bio_sarge"		"Joseph Conrad ist ein harter, erfahrener Soldat. Er ist einer der wenigen, die eine groß angelegte Invasion des Schwarms überlebt haben, eine Erfahrung, die ihn zu dem Mann gemacht hat, der er heute ist. Er ist ein starker Kommandant, der mit gutem Beispiel vorangeht und auch verbale Tritte in den Hintern nicht scheut. Er kennt sein Team gut und setzt sich dafür ein, dass sie ihr Bestes geben."
```

Comment:
if marine bios are changed, they should be changed in the item descriptions (https://github.com/ReactiveDrop/reactivedrop_translations/blob/master/community/inventory_service/item-schema-strange-equipment.json) as well

Reply:
Apologies, didn't know that; will update them in a minute.

Reply:
There's a lot of little things like that and it's my job to remember them 😁